pkg/proxy/ipvs/proxier.go

```diff
@@ -341,6 +343,7 @@ func NewProxier(ipt utiliptables.Interface,
 	healthzServer healthcheck.ProxierHealthUpdater,
 	scheduler string,
 	nodePortAddresses []string,
+	handle KernelHandler,
```

Comment:
nit: `kernelHandler`

Reply:
Does this mean we should create a new parameter of type KernelHandler? I was reusing the [existing KernelHandler struct](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/proxier.go#L563-L567) like the [current GetKernelFunction & CanUseIPVSProxier](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/proxier.go#L662-L677) .

Reply:
Sorry I wasn't clear, was referring to the param name `handle` -> `kernelHandler`